src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PrinterSettings.Windows.cs

```diff
@@ -644,61 +644,31 @@ public Graphics CreateMeasurementGraphics(PageSettings pageSettings, bool honorO
             return g;
         }
 
-
         // Create a PRINTDLG with a few useful defaults.
         // Try to keep this consistent with PrintDialog.CreatePRINTDLG.
-        private static Interop.Comdlg32.PRINTDLGX86 CreatePRINTDLGX86()
-        {
-            Interop.Comdlg32.PRINTDLGX86 data = new Interop.Comdlg32.PRINTDLGX86();
-            data.lStructSize = Marshal.SizeOf(typeof(Interop.Comdlg32.PRINTDLGX86));
-            data.hwndOwner = IntPtr.Zero;
-            data.hDevMode = IntPtr.Zero;
-            data.hDevNames = IntPtr.Zero;
-            data.Flags = 0;
-            data.hwndOwner = IntPtr.Zero;
-            data.hDC = IntPtr.Zero;
+        private static unsafe Interop.Comdlg32.PRINTDLGX86 CreatePRINTDLGX86()
+        {
+            Interop.Comdlg32.PRINTDLGX86 data = default;
+            data.lStructSize = sizeof(Interop.Comdlg32.PRINTDLGX86);
             data.nFromPage = 1;
             data.nToPage = 1;
             data.nMinPage = 0;
             data.nMaxPage = 9999;
             data.nCopies = 1;
-            data.hInstance = IntPtr.Zero;
-            data.lCustData = IntPtr.Zero;
-            data.lpfnPrintHook = IntPtr.Zero;
-            data.lpfnSetupHook = IntPtr.Zero;
-            data.lpPrintTemplateName = null;
-            data.lpSetupTemplateName = null;
-            data.hPrintTemplate = IntPtr.Zero;
-            data.hSetupTemplate = IntPtr.Zero;
             return data;
         }
 
-
         // Create a PRINTDLG with a few useful defaults.
         // Try to keep this consistent with PrintDialog.CreatePRINTDLG.
-        private static Interop.Comdlg32.PRINTDLG CreatePRINTDLG()
-        {
-            Interop.Comdlg32.PRINTDLG data = new Interop.Comdlg32.PRINTDLG();
-            data.lStructSize = Marshal.SizeOf(typeof(Interop.Comdlg32.PRINTDLG));
-            data.hwndOwner = IntPtr.Zero;
-            data.hDevMode = IntPtr.Zero;
-            data.hDevNames = IntPtr.Zero;
-            data.Flags = 0;
-            data.hwndOwner = IntPtr.Zero;
-            data.hDC = IntPtr.Zero;
+        private static unsafe Interop.Comdlg32.PRINTDLG CreatePRINTDLG()
```

Comment:
@jkotas this function called in 2 places. Can I inline it, since this is now copying structures.

Reply:
I thin I can use `ref` return values here.